src/gbt_convertors.pyx

```diff
@@ -180,8 +180,17 @@ class Node:
             right_child = None
 
         is_leaf = "leaf_value" in tree
+        if is_leaf:
+            if "leaf_count" in tree:
+                cover=tree["leaf_count"]
+            else:
+                cover=0
+        else:
+            cover=tree["internal_count"]
+
+        value = {tree["leaf_value"] if is_leaf else tree["threshold"]}
```

Comment:
```suggestion
        # get cover and value for leaf nodes or internal nodes
        cover = tree.get("leaf_count", 0) or tree.get("internal_count", 0)
        value = tree.get("leaf_value", 0) or tree.get("threshold", 0)
```

I would just write it a bit more compact. Should work for `.pyx`, no?

Reply:
I have just checked this version. It works fine